ow_faults_injection/cfg/Faults.cfg

```diff
@@ -48,8 +48,8 @@ ft_sensor_faults.add("signal_noise_failure", double_t, 0,
 
 # POWER FAULTS
 power_faults = gen.add_group("power_faults")
-power_faults.add("low_state_of_charge_power_failure", bool_t,   0, "Low state of charge power failure", False)
-power_faults.add("instantaneous_capacity_loss_power_failure", bool_t,   0, "Instantaneous capcity loss power failure", False)
-power_faults.add("thermal_power_failure", bool_t,   0, "Thermal power failure", False)
+power_faults.add("high_power_draw_power_failure",   double_t, 0,
+    "High power draw power failure", 0.0, 0.0, 30.0)
```

Comment:
Just "high power draw" would be better, since the power draw itself isn't a failure -- it's potential effects are.

Reply:
Sounds good. I was naming them based on the previous power fault naming scheme (where 'power_failure' was appended to every name) even if it seemed a little convoluted to name something 'high power draw power failure'.

Having the minimum value set to 0.0 was a holdover from a previous iteration before there was the 'enable/disable' checkbox for power draw activation. Will fix that too.

Reply:
As a heads up, the code will not build if I don't follow the set naming conventions. Attempting to name the slider "High power draw" instead of "high_power_draw" caused an error. I updated the names and mouse-hover labels within the naming convention limitations.